components/emoji_picker/emoji_picker.jsx

```diff
@@ -350,27 +347,27 @@ export default class EmojiPicker extends React.PureComponent {
         case 'ArrowRight':
             e.preventDefault();
             this.selectNextEmoji();
-            this.searchInput.focus();
+            this.searchInputRef?.current?.focus();
             break;
         case 'ArrowLeft':
             e.preventDefault();
             this.selectPrevEmoji();
-            this.searchInput.focus();
+            this.searchInputRef?.current?.focus();
             break;
         case 'ArrowUp':
             e.preventDefault();
             this.selectPrevEmoji(EMOJI_PER_ROW);
-            this.searchInput.focus();
+            this.searchInputRef?.current?.focus();
             break;
         case 'ArrowDown':
             e.preventDefault();
             this.selectNextEmoji(EMOJI_PER_ROW);
-            this.searchInput.focus();
+            this.searchInputRef?.current?.focus();
             break;
         }
     }
 
-    handleKeyDown = (e) => {
+    handleSearchKeyDown = (e) => {
```

Comment:
I think all this logic can be moved in the new component, probably we would need to remove the onKeyDown and add something like `onSelectEmoji`.